.changeset/seven-balloons-attack.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
TODO: revamp this changeset since `ActorImpl` is no longer a thing and we require `logic` (`{ 
src: string; id?: string; logic: AnyActorLogic }`)